docs/source/AdministratorGuide/DIRACInstall/index.rst

```diff
@@ -0,0 +1,126 @@
+.. _diracinstall:
+
+==========================================
```

Comment:
This whole doc file is a bit disconnected from the others, and present some repetitions wrt to what is already there. It needs to be better included.

Reply:
I do not know where to put and I think this document should be one of the first thing to read before starting to use DIRAC.

Reply:
Then just put it back to the file as it was before. We'll make sure it's correctly displayed via Sphinx.

Reply:
okay. Probably it is the best place. I put back the doc